scripts/skills/actives/rf_command_skill.nut

```diff
@@ -0,0 +1,85 @@
+this.rf_command_skill <- this.inherit("scripts/skills/skill", {
+	m = {
+		ActionPointsRecovered = 1
+	},
+
+	function create()
+	{
+		this.m.ID = "actives.rf_command";
+		this.m.Name = "Command";
+		this.m.Description = "You shall do it!\nThe targeted ally moves forward in the current turnorder and recovers 1 Action Point. \nCan not be used on fleeing or stunned allies.";
```

Comment:
typo: `turn order` missing a space. Imo, the mechanical information should be in the `getTooltip` and not in the Description. A good description for this skill could be `Command an ally to act now!`. Then in the `getTooltip` you show the mechanical behavior i.e. 1. the targeted ally moves to the next spot in the turn order, 2. the targeted ally recovers 1 Action Point. 3. Cannot be used on animals, or stunned or fleeing allies.`